util/site/build-docs.sh

```diff
@@ -161,19 +161,22 @@ buildSite () {
     echo "Doxygen build complete."
 
     # shellcheck disable=SC2086
-    ${book_env}  ./bazelisk.sh run @crate_index//:mdbook__mdbook -- ${mdbook_args}
+    ${book_env} ./bazelisk.sh run --experimental_convenience_symlinks=ignore @crate_index//:mdbook__mdbook -- ${mdbook_args}
     # shellcheck disable=SC2086
-    ${book_guides_env} ./bazelisk.sh run @crate_index//:mdbook__mdbook -- ${mdbook_guides_args}
+    ${book_guides_env} ./bazelisk.sh run --experimental_convenience_symlinks=ignore @crate_index//:mdbook__mdbook -- ${mdbook_guides_args}
     # shellcheck disable=SC2086
     hugo ${hugo_args}
 
     # Build Rust Documentation
-    rust_doc_dir="${build_dir}/gen/rustdoc/"
-    mkdir -p "$rust_doc_dir"
-    ./bazelisk.sh build sw/host/opentitanlib:opentitanlib_doc
-    cp -rf bazel-out/k8-fastbuild/bin/sw/host/opentitanlib/opentitanlib_doc.rustdoc/* "$rust_doc_dir"
+    local rustdoc_dir="${build_dir}/gen/rustdoc/"
+    mkdir -p "${rustdoc_dir}"
+    local target_rustdoc target_rustdoc_output_path
+    target_rustdoc="sw/host/opentitanlib:opentitanlib_doc"
+    target_rustdoc_output_path="$(./bazelisk.sh info output_path 2>/dev/null)/k8-fastbuild/bin/$(echo ${target_rustdoc} | tr ':' '/').rustdoc" #TODO : get this using cquery
```

Comment:
Nit: I would find this easier to read

```suggestion
    bazel_out="$(./bazelisk.sh info output_path 2>/dev/null)"
     #TODO : get the target's path using cquery
    target_rustdoc_output_path="$bazel_out/k8-fastbuild/bin/$(echo ${target_rustdoc} | tr ':' '/').rustdoc"
```

Reply:
Done